### ISSUE TITLE
Cannot change autostart in config

### ISSUE THREAD

**Issue by Seb33300** [tetranz/select2entity-bundle#63]
Hello,

I would like to change the autostart option globally, so I added it to the config.yml:

```
tetranz_select2_entity:
    auto_start: false
```

But:

> Unrecognized option "auto_start" under "tetranz_select2_entity"

Is it normal?

**Comment by tetranz** [comment]
Yeah, that's "normal" because autostart is not a global setting so for now you'll have to set it per instance.

It probably should be global so I'll try to get to that when I have time but anyone is welcome to submit a P.R.
